packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReactElement } from 'react';
+import type { SubmitHandler, Validate } from 'react-hook-form';
+
+import BackgroundLayer from '../../common/BackgroundLayer';
+import FormPageLayout from '../../common/FormPageLayout';
+import AdminInfoForm from '../../forms/AdminInfoForm';
+
+type CloudAdminInfoPageProps = {
+  title: string;
+  subtitle: string;
+  currentStep: number;
+  stepCount: number;
+  validateUsername: Validate<string>;
+  validateEmail: Validate<string>;
+  passwordRulesHint: string;
+  keepPosted?: boolean;
+  validatePassword: Validate<string>;
+  onSubmit: SubmitHandler<{
+    fullname: string;
+    username: string;
+    companyEmail: string;
+    password: string;
+  }>;
+};
+
+const CloudAdminInfoPage = (props: CloudAdminInfoPageProps): ReactElement => (
```

Comment:
What is different in this component in comparison with `AdminInfoPage`? It's not very clear for me.

Reply:
As we discuss previously any difference that justifies a new component, the fix will remove this one!